google/cloud/dlp_v2/gapic/dlp_service_client.py

```diff
@@ -40,8 +40,10 @@
 from google.protobuf import empty_pb2
 from google.protobuf import field_mask_pb2
 
-
-_GAPIC_LIBRARY_VERSION = pkg_resources.get_distribution("google-cloud-dlp").version
+try:
+  _GAPIC_LIBRARY_VERSION = pkg_resources.get_distribution("google-cloud-dlp").version
+except pkg_resources.DistributionNotFound:
+  _GAPIC_LIBRARY_VERSION = "0.14.0"
```

Comment:
👍 thanks for the work-around. Might make sense to set the version in dlp.py or somewhere more visible?

Reply:
Also, if you're going to bump the version, should you match that in setup.py?